examples/PySDM_examples/deJong_Mackay_et_al_2023/settings_0D.py

```diff
@@ -37,10 +37,10 @@ def __init__(
         self._steps = [0]
         self.kernel = Geometric()
         self.coal_eff = Berry1967()
-        self.fragmentation = fragmentation or ExponFrag(scale=self.frag_scale)
+        self.fragmentation = fragmentation or Exponential(scale=self.frag_scale)
         self.vmin = 0.0
         self.break_eff = ConstEb(1.0)  # no "bouncing"
-        self.spectrum = Exponential(norm_factor=self.norm_factor, scale=self.X0)
+        self.spectrum = spectra.Exponential(norm_factor=self.norm_factor, scale=self.X0)
```

Comment:
just a nitpick, but I think we should use module names in both cases: `breakup_fragmentations.Exponential` and `spectra.Exponential`